HelpSource/Classes/Ndef.schelp

```diff
@@ -161,6 +161,97 @@ Ndef(\a) <<>.phase nil;	// unmap
 Ndef.clear(3);		// fade out and clear all Ndefs
 ::
 
+subsection::Embedding multi-channel Patterns, playing Streams in parallel
+
+Controlling multi-channeled sequenced streams and having independent control over filtering and node ordering is a difficult topic in SuperCollider. However, using Ndefs (or their superclass link::Classes/NodeProxy:: or a link::Classes/ProxySpace::) may provide a convenient solution.
+
+code::
+// a SynthDef, creating single-channel grain when instantiated
+(
+SynthDef(\grain, { |out=0, freq=300, amp=0.3|
+	OffsetOut.ar(out, Pulse.ar(freq) * EnvGen.kr(Env.perc, doneAction: 2) * amp)
+}).add;
+)
+
+// number of channels
+~numChans = 5;
+
+// values in a Pattern may be set in various ways
+// here we use control buses, except for \dur which
+// doesn't accept a control bus in parallel playing streams
+// therefore we use PatternProxies
+~durs = ~numChans.collect({ |i| PatternProxy(0.5 + (i/10)) });
+
+// other parameters could as well be controlled in PatternProxies,
+// yet, control buses are convenient either
+~freqs = Bus.control(s, ~numChans);
+~freqs.setn(Array.geom(~numChans, 300, 1.1));
+~amps = Bus.control(s, ~numChans);
+~amps.setn(0.2!~numChans);
+
+// the Pattern: a Ppar holding one Pbind for each channel,
+// all wrapped in a Pdef
+(
+Pdef(\ppar,
+    Ppar({ |i|
+        Pbind(
+            \instrument, \grain,
+            // we only set a single channel
+            \dur, ~durs[i],
```

Comment:
couldn't you use Ndefs instead of busses? Like this:
```
Ndef(\freqs, Array.geom(~numChans, 300, 1.1));
Ndef(\amps, 0.2 ! ~numChans);


// the Pattern: a Ppar holding one Pbind for each channel,
// all wrapped in a Pdef
(
Pdef(\ppar,
    Ppar({ |i|
        Pbind(
            \instrument, \grain,
            // we only set a single channel
            \dur, ~durs[i],
			\freq, Ndef(\freqs),
			\amp, Ndef(\amps),
            // the Pattern will play to a yet unknown private bus
            // we only want to make sure the offset is right
            \channelOffset, i,
        )
    }!~numChans)
)
)
```